json/emtech/social/twitter_emergingtech-index-pattern.json

```diff
@@ -3,7 +3,7 @@
                 "release_date": "2018-09-20T12:55:30.119472",
```

Comment:
Please update this date too in order to track the version of the index pattern when deployed.

More info about this can be found at: https://github.com/chaoss/grimoirelab-kidash/commit/a0e6b758f1ce37e15f0decbc69589b4ed05971d9

Reply:
My apologies for the late reply. Was having some issues setting up kidash.